### PR TITLE
Add my subdomain

### PR DESCRIPTION
**Pull request by ni5arga**
@sampoder

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -232,7 +232,7 @@ _vercel:
     - vc-domain-verify=epochba.hackclub.com,705735e32a03c39fdd26
     - vc-domain-verify=lghs.hackclub.com,f33560bd22273ee8df77
     - vc-domain-verify=linkscape.hackclub.com,c31824abf58faa7a0160
-
+    - vc-domain-verify=dsc.hackclub.com,151b54b3f4af4d093d8d.
```
Comment (sampoder) [inline hackclub.com.yaml:235]:
```suggestion
    - vc-domain-verify=dsc.hackclub.com,151b54b3f4af4d093d8d.
    
```

File: hackclub.com.yaml [hunk 2]
```diff
@@ -697,6 +697,13 @@ drive:
   ttl: 1
   type: CNAME
   value: ghs.googlehosted.com.
+dsc:
+- ttl: 1
+  type: CNAME
+  value: cname.vercel-dns.com.
+  ttl: 1
+  type: TXT
+  value: vc-domain-verify=dsc.hackclub.com,151b54b3f4af4d093d8d.
```
Comment (sampoder) [inline hackclub.com.yaml:706]:
```suggestion
```

File: hackclub.com.yaml (hunk 1 above)
Comment (ni5arga) [reply hackclub.com.yaml:235]: I'll fix

**Comment by ni5arga** [comment]
Fixed.

**Comment by ni5arga** [comment]
Thank you

**Comment by ni5arga** [comment]
@sampoder whenever I try to visit the website I get a SSL error.

```
dsc.hackclub.com sent an invalid response.
ERR_SSL_PROTOCOL_ERROR
```

**Pull request merged**